docs/source/building.rst

```diff
@@ -0,0 +1,123 @@
+.. _building:
+
+Building PYJNIus
+================
+
+Building PyJNIus depends on `Cython <http://cython.org/>`_ and the `Java
+<http://www.oracle.com/javase>`_ Development Kit (includes the Java Runtime
+Environment).
+
+
+Building on GNU/Linux distributions
+-----------------------------------
+
+You need the GNU Compiler Collection (GCC), the JDK installed (openjdk
+will do), and Cython. Then, just type::
+
+    sudo python setup.py install
```

Comment:
Ouch! We really want to suggest using `sudo` ? 😬

And, maybe is better to suggest using `pip install .` ?